CSS properties: intro sentence in values section part 3 (#44936)
mdn/content: files/en-us/web/css/reference/properties/color/index.md

color: unset;
```

The `color` property is specified as a single {{cssxref("&lt;color&gt;")}} value.

> Note that the value must be a uniform color. It can't be a {{cssxref("gradient")}}, which is actually a type of {{cssxref("image")}}.

Reviewer: can we retain this Note? looks like important guidance

Author: this was possiblyimportant years ago, but people understnd that color and gradient are different.
we don't have it on other props that accept color

> This property is specified as one {{cssxref("&lt;color&gt;")}} value:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
